lke_clusters.go

```diff
@@ -239,6 +239,13 @@ func (c *Client) GetLKEClusterKubeconfig(ctx context.Context, clusterID int) (*L
 	return response, nil
 }
 
+// DeleteLKEClusterKubeconfig deletes the Kubeconfig for the LKE Cluster specified
+func (c *Client) DeleteLKEClusterKubeconfig(ctx context.Context, clusterID int) error {
+	e := formatAPIPath("lke/clusters/%d/kubeconfig", clusterID)
+	err := doDELETERequest(ctx, c, e)
+	return err
```

Comment:
```suggestion
	return doDELETERequest(ctx, c, e)
```

Reply:
Yeah you're right. It looks like this can be done in many of the calls to doDELETERequest across the repo. I can make a separate PR to fix it everywhere.

Reply:
Fixed it in the files touched in this PR, we can fix the rest in a separate one 👍